movement/io/load_poses.py

```diff
@@ -678,3 +679,102 @@ def _ds_from_valid_data(data: ValidPosesDataset) -> xr.Dataset:
             "source_file": None,
         },
     )
+
+
+def from_mmp_file(
+    file_path: Union[Path, str], fps: Optional[float] = None
+) -> xr.Dataset:
+    """Load pose tracking data from a JSON file with multiple individuals.
```

Comment:
As of [PR 194](https://github.com/neuroinformatics-unit/movement/pull/194) we changed the phrasing of such docstrings:

```suggestion
    """Create a ``movement`` dataset from an MMPose .json file
```